setup-renv/action.yaml

```diff
@@ -2,6 +2,8 @@ name: 'setup-renv'
 description: 'Action to setup renv and install R dependencies in the lockfile'
 author: 'Jim Hester'
 inputs:
+  profile:
+    description: 'The renv profile that should be activated. Forwarded to `renv::activate()`. Note that it often needs to be quoted in YAML, see the README for details.'
```

Comment:
Do we need a default here? Also can you pls mention that this is an R expression. Thanks!

Reply:
No, no need for a default, as demonstrated in the [test repo](https://github.com/Bisaloo/testrenvprofile/blob/main/.github/workflows/test.yml).

If I understand correctly, an empty value in YAML corresponds to `NULL` in R, i.e., the default value of `profile` in `renv::activate()` so it's all good.